### PR TITLE
eq-8 Response Validation

### PR DESCRIPTION
**Pull request by hamishtaplin**
## Changes
- integrated response validation with templates, showing errors
- updated schema with latest MCI content (also reformatted, hence the diff looking like the whole things has changed)
- updated some styling as per latest prototypes
## Test
- `npm run compile`
- test numeric validation works by submitting survey with invalid (eg. non-numeric) data
- test required field validation by omitting data

### CODE REVIEW

File: app/data/mci.json [hunk 1]
```diff
@@ -1,186 +1,143 @@
...
+      }, {
+        "id": "d9d25a21-41b8-42b2-ac7b-fb21b1036d71",
+        "title": "Section 2",
+        "questions": [{
+          "id": "eedb9f9f-2f2e-41f4-9186-ba7110d9e6a4",
+          "title": "Commodities - Retail Turnover",
+          "description": "<p>- You should enter figures for the reporting period stated above<br>- You should round your figures to the nearest (£) pound</p><h4>Include</h4><ul><li>VAT</li><li>Internet Sales</li></ul>",
+          "responses": [{
+            "id": "bb8168e6-2272-450d-b5a7-d3170508efb2",
+            "q_code": "117",
```
Comment (warrenbailey) [inline app/data/mci.json]: You've lost the updated q-codes

**Comment by warrenbailey** [comment]
When updating the schema you've reverted to the old q_codes. We need to put the new ones back in. :-1:

File: app/styles/components/forms/_input.scss [hunk 2]
```diff
@@ -10,15 +10,15 @@ $input-padding: 0.6rem;
...
-    box-shadow: 0 0 0 1px tint($color-blue, 50%);
-  }
-  &.has-error {
-    border-color: $color-red;
-  }
+  max-width: 40rem;
+  // &:focus {
+  //   outline: none;
+  //   border: 1px solid tint($color-blue, 50%);
+  //   box-shadow: 0 0 0 1px tint($color-blue, 50%);
```
Comment (warrenbailey) [inline app/styles/components/forms/_input.scss]: Do we need this? Can it be removed?

**Comment by hamishtaplin** [comment]
Schema should be ok again now.

**Comment by warrenbailey** [comment]
LGTM :+1:

**Pull request merged**